account_multicompany_code/models/account_journal.py

```diff
@@ -0,0 +1,12 @@
+from odoo import api, models
+
+
+class AccountJournal(models.Model):
+    _inherit = "account.journal"
+
+    @api.depends("company_id.code")
+    def _compute_display_name(self):
+        super()._compute_display_name()
```

Comment:
Store and return result (this is a lint)

Reply:
done